examples/deployment/assets/gold-john-bob.mlsig

```diff
@@ -0,0 +1,33 @@
+// This file was created using goal by doing
+//   - create a new multisignature account (goal account multisig new ..)
+//   - create logic signature and sign the logic signature using `goal clerk multisig signprogram..`
+//   - output of Decompiling the logic signature is pasted in this file
+// reference: https://developer.algorand.org/docs/features/asc1/stateless/walkthrough/#creating-a-multi-signature-delegated-logic-sig
```

Comment:
Why do we need to decompile the file?

Reply:
Updated the comment. We are disassembling the raw binary lsig to use it better.